src/shipping/shipping-table-rate.md

```diff
@@ -153,23 +157,23 @@ This example uses the Weight v. Destination condition to create different shippi
 |AUS|WA|*|0|19.95|
 |AUS|*|*|9|29.95|
 |AUS|*|*|0|9.95|
-{:style="table-layout:auto"}
 
-### Example 3: Restrict Free Shipping to the Continental United States
+### Example 3: Restrict free shipping to the continental United States
 
 1. Create a `tablerates.csv` file that includes all the state destinations to which you are willing to provide free shipping.
 
 1. Complete the table rate configuration with the following settings:
 
-    | Setting | Value |
-    |----------
-    | Condition | Price v. Destination |
-    | Method Name | Free Shipping |
-    | Ship to Applicable Countries | Specific Countries |
-    | Ship to Specific Countries | Select only United States |
-    | Show method if not applicable | No |
-    {:style="table-layout:auto"}
+   | Setting | Value |
+   |----------
```

Comment:
is this missing the second column and end hashes?

Reply:
It still renders correctly, but better practice to add them